.gitignore

```diff
@@ -1,8 +1,18 @@
-# OS X
-.DS_Store
```

Comment:
@boda-taljo i am not in favor of removing finder metadata file `.DS_Store` from `.gitignore` since the chance of accidentally pushing this file will be there, let's do not remove it.